
string station_name  # station name ... ex: A, B , C , ..

int32 station_priority # priority of the station numbered from 1:3 where 3 is the heighest priority

Position station_location # 3d position of the station

